apis/Google.Bigquery.V2/Google.Bigquery.V2/BigqueryParameter.cs

```diff
@@ -55,20 +55,13 @@ namespace Google.Bigquery.V2
     ///   which will be inferred from the value's element type if not otherwise specified.</description></item>
     /// </list>
     /// <para>
-    /// If the parameter type is null, it is inferred from the value. In most cases that is obvious from the above
-    /// table, but for date and time values the process is more complicated. A <see cref="TimeSpan"/> value is
-    /// always assumed to be a <see cref="BigqueryParameterType.Time"/>, and a <see cref="DateTimeOffset"/> value is
-    /// always assumed to be a <see cref="BigqueryParameterType.Timestamp"/>. A <see cref="DateTime"/> value is assumed
-    /// to be <see cref="BigqueryParameterType.DateTime"/> if its <see cref="DateTime.Kind"/> is <see cref="DateTimeKind.Unspecified"/>
-    /// or <see cref="DateTimeKind.Local"/>, but <see cref="BigqueryParameterType.Timestamp"/> if its <c>Kind</c> is
-    /// <see cref="DateTimeKind.Utc"/>.
+    /// If the parameter type is null, it is inferred from the value. A <see cref="TimeSpan"/> value is
+    /// always assumed to be a <see cref="BigqueryParameterType.Time"/>, a <see cref="DateTimeOffset"/> value is
+    /// always assumed to be a <see cref="BigqueryParameterType.Timestamp"/>, and a <see cref="DateTime"/> value is assumed
```

Comment:
Hmm, it also now seems slightly odd that DateTimeOffset is inferred differently to DateTime.
I think I'd prefer they both infer to DateTime; from a C# perspective these are both representing basically the same thing, so it seems odd they infer the different Datastore things.
What do you think? (Sorry, should have brought this up earlier)